scripts/publish.sh

```diff
@@ -88,13 +88,13 @@ npm run build:release
 echo "Ran publish build."
 
 echo "Making a $VERSION version..."
-# TODO: Remove the following command.
-#   npm version command had previously failed claiming unclean git repo, and we don't know why.
-echo "DEBUG: Running git status to show dirty files..."
-git status
-npm version $VERSION
+if [[ $PRE_RELEASE == "" ]]; then
```

Comment:
Should we switch this if statement around since below we make the inverted check? Might be easier readability if they're all the same direction

Reply:
goooood call.